scripts/set_xtensa_params.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
```

Comment:
If you drop this line then it may catch people making the mistake of NOT `source`ing this script.

shellcheck will complain but ignore that.